SQL/scripts.sql

```diff
@@ -1,7 +1,7 @@
 create Schema 2024_batch_jdbc;
```

Comment:
change your column names from camelcase to upper case with separate underscore.

Reply:
ok sir